twitter.html

```diff
@@ -0,0 +1,327 @@
+            <a class="nav" href="#">
+                <h2>Bookmarks</h2>
+            </a>
+
+            <img src="./images/SVG's/file-06-svgrepo-com.svg" alt="" width="30px">
+            <a class="nav" href="#">
+                <h2>Lists</h2>
+            </a>
+
+            <img src="./images/SVG's/profile-round-1342-svgrepo-com.svg" alt="" width="30px">
+            <a class="nav" href="#">
+                <h2>Profile</h2>
+            </a>
+
+            <img src="./images/SVG's/more-horizontal-circle-svgrepo-com.svg" alt="" width="30px">
+            <a class="nav" href="#">
+                <h2>More</h2>
+            </a>
+
+            <button class="tweet_button" type="button">Tweet</button>
+
+            <div class="profile-left-menu">
+                <img class="profile-photo" src="/images/IMG-20230317-WA0007.jpg" width="100px" alt="profile-photo">
+                <div class="profile">
+                    <p>Lukas</p>
+                    <p>@PeachyOmnivore</p>
+                </div>
+                <img class="hoz-dots" src="./images/SVG's/dots-horizontal-svgrepo-com.svg" width="30px" alt="">
+            </div>
+
+        </div>
+
+<!-- MAIN -->
+        <main class="main">
+            <header>
+                <div class="home">
+                    <h1>Home</h1><img src="./images/SVG's/stars-02-svgrepo-com.svg" alt="3 stars" width="30px">
+                </div>
+                <div class="post-bar"><img class="profile-photo" src="./images/IMG-20230317-WA0007.jpg" alt="">
+                    <textarea name="post-area" id="post-area" cols="60" rows="3">What's Happening?</textarea>
+                </div>
+                <div class="post-buttons">
+                    <button class="post-nav-button"><img class="post-nav"
+                            src="./images/SVG's/posting-navigation/picture-svgrepo-com.svg" alt="pictures"></button>
+                    <button class="post-nav-button"><img class="post-nav"
+                            src="./images/SVG's/posting-navigation/gif-svgrepo-com.svg" alt=""></button>
+                    <button class="post-nav-button"><img class="post-nav"
+                            src="./images/SVG's/posting-navigation/stats-square-up-svgrepo-com.svg" alt=""></button>
+                    <button class="post-nav-button"><img class="post-nav"
+                            src="./images/SVG's/posting-navigation/smiley-svgrepo-com.svg" alt=""></button>
+                    <button class="post-nav-button"><img class="post-nav"
+                            src="./images/SVG's/posting-navigation/calender-svgrepo-com.svg" alt=""></button>
+                    <button class="tweet-button2">TWEET</button>
+                </div>
+            </header>
+            <div class="spacer">
+            </div>
+            <div class="feed">
+                <!-- POST ONE -->
+                <div class="feed-post-grid">
```

Comment:
Descriptive class names like this make it a lot easier to understand the purpose of each section so well done on this!